README.md

```diff
@@ -0,0 +1,52 @@
+# Cluster Version Operator (CVO)
+
+## Building and Publishing CVO
+
+```sh
+./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.go
+```
+
+This builds image using `podman` locally using following command:
+
+```sh
+podman build -t "cluster-version-operator:${VERSION}" -f Dockerfile
+```
+
+And then pushes the image to 2 sources:
+
+```sh
+podman push "cluster-version-operator:${VERSION}" "${REPO}/origin-cluster-version-operator:${VERSION}"
+podman push "cluster-version-operator:${VERSION}" "${REPO}/origin-cluster-version-operator:latest"
+```
+
+## Building release image using local CVO
+
+1. Make sure you have `oc` binary from https://github.com/openshift/origin master as it requires `adm release` subcommand.
+
+2. Run the following command to create release-image at `docker.io/abhinavdahiya/origin-release:latest`:
+
+```sh
+oc adm release new -n openshift --server https://api.ci.openshift.org \
+    --from-image-stream=origin-v4.0 \
+    --to-image-base=docker.io/abhinavdahiya/origin-cluster-version-operator:latest \
+    --to-image docker.io/abhinavdahiya/origin-release:latest
+```
+
+## Using CVO to render the release-payload locally
+
+1. Run the following command to get render the release-payload contents to `/tmp/cvo`
+
+```sh
+podman run --rm -ti \
+    -v /tmp/cvo:/tmp/cvo:z \
+    <release image> \
+        render \
+        --output-dir=/tmp/cvo \
+        --release-image="<release image>"
```

Comment:
Do we have real location we can plug in here?

Reply:
added another line below with options.